orttraining/orttraining/test/python/launch_test.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import sys
+import argparse
+import subprocess
+
+import logging
+
+logging.basicConfig(
+    format="%(asctime)s %(name)s [%(levelname)s] - %(message)s",
+    level=logging.DEBUG)
+
+log = logging.getLogger("Build")
+
+def run_subprocess(args, cwd=None, env={}):
```

Comment:
can this common function be refactored out and reused?

Reply:
refactored to a common method called by multiple scripts.

Reply:
refactored into a common method called from multiple scripts